src/C4 정렬/Q23 JS1 국영수.js

```diff
@@ -0,0 +1,40 @@
+function solution(n, point) {
+  let scoreArray = [];
+  point.forEach((val) => {
+    const [name, k, e, m] = val.split(" ");
+    console.log(k);
+    scoreArray.push({
+      name,
+      korean: Number(k),
+      english: Number(e),
+      math: Number(m),
+    });
+  });
+  scoreArray.sort((a, b) => {
+    if (a.korean > b.korean) return -1;
+    if (a.korean < b.korean) return 1;
+    if (a.english > b.english) return 1;
+    if (a.english < b.english) return -1;
+    if (a.math > b.math) return -1;
+    if (a.math < b.math) return 1;
+  });
```

Comment:
```suggestion
    if (a.korean > b.korean) return -1;
    if (a.korean < b.korean) return 1;
    if (a.english > b.english) return 1;
    if (a.english < b.english) return -1;
    if (a.math > b.math) return -1;
    if (a.math < b.math) return 1;
    return 0;
  });
```

마지막에 `return 0` 까지 해주면 best이지 않을까요~~

Reply:
모두 같을 때 반환값을 제외했네요 좋은 의견 감사합니다